examples/evm/call-contract-with-token/CallContractWithToken.sol

```diff
@@ -6,13 +6,35 @@ import { IAxelarGateway } from '@axelar-network/axelar-gmp-sdk-solidity/contract
 import { IERC20 } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IERC20.sol';
 import { IAxelarGasService } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGasService.sol';
 
-contract DistributionExecutable is AxelarExecutable {
+
+/**
+ * @title Call Contract With Token 
+ * @author Axelar 
```

Comment:
Do we need these author and title notes or should we leave them out?

Reply:
I wanted to have inline docs for our functions, so figured to have this as well for consistency, can remove if you want though

Reply:
@benjamin852 definitely remove author